app/models/form/lettings/subsections/property_information.rb

```diff
@@ -10,8 +10,8 @@ def pages
     @pages ||= [
       uprn_questions,
       Form::Lettings::Pages::PropertyLocalAuthority.new(nil, nil, self),
-      Form::Lettings::Pages::MinRentValueCheck.new("local_authority_min_rent_value_check", nil, self, check_answers_card_number: nil),
-      Form::Lettings::Pages::MaxRentValueCheck.new("local_authority_max_rent_value_check", nil, self, check_answers_card_number: nil),
+      Form::Lettings::Pages::MinRentValueCheck.new("local_authority_min_rent_value_check", nil, self, check_answers_card_number: nil), # Soft validation
+      Form::Lettings::Pages::MaxRentValueCheck.new("local_authority_max_rent_value_check", nil, self, check_answers_card_number: nil), # Soft validation
```

Comment:
Did you want to leave the comments here?

Reply:
I thought I'd mark the questions I left out for the next person who works on the soft validations, I can remove if its not helpful.